cmd/autoconfigbrancher/main.go

```diff
@@ -195,6 +195,22 @@ func main() {
 				"--prune=true",
 			},
 		},
+		{
+			command: "/usr/bin/registry-replacer",
```

Comment:
This needs to run first, before the config brancher

Reply:
updated